xds/src/test/java/io/grpc/xds/BootstrapperTest.java

```diff
@@ -78,7 +78,7 @@ public void parseBootstrap_validData_singleXdsServer() throws IOException {
     assertThat(serverInfo.getChannelCredentials().get(1).getConfig()).isNull();
     assertThat(serverInfo.getChannelCredentials().get(2).getType()).isEqualTo("google_default");
     assertThat(serverInfo.getChannelCredentials().get(2).getConfig()).isNull();
-    assertThat(info.getNode()).isEqualTo(
+    assertThat(info.getNode().toEnvoyProtoNodeV2()).isEqualTo(
```

Comment:
This way of unit testing is wrong. The output of parsing the bootstrap file has changed, you should verify the direct output (aka, `EnvoyProtoData.Node`) instead of converting it to a proto message. Logics of converting `EnvoyProtoData.Node` to a proto message should be covered in its own unit tests (which we currently do not have).

Reply:
Fixed.